addons/medical/functions/fnc_handleDamage.sqf

```diff
@@ -77,13 +79,15 @@ _minLethalDamage = if (_typeIndex >= 0) then {
     0.01
 };
 
-if (vehicle _unit != _unit && {!(vehicle _unit isKindOf "StaticWeapon")} && {isNull _shooter} && {_projectile == ""} && {_selection == ""}) then {
+private _vehicle = vehicle _unit;
+private _effectiveSelectionName = _selection;
+if ((_vehicle != _unit) && {!(_vehicle isKindOf "StaticWeapon")} && {_shooter in [objNull, driver _vehicle, _vehicle]} && {_projectile == ""} && {_selection == ""}) then {
     if (GVAR(enableVehicleCrashes)) then {
-        _selection = GVAR(SELECTIONS) select (floor(random(count GVAR(SELECTIONS))));
+        _effectiveSelectionName = _this select 1; //pull random selection from HDC
```

Comment:
On crash, handle damage caching will convert the `""` into a random selection,
I think we want the same value for passing to `FUNC(determineIfFatal)`
But we don't want to overwrite `_selection` for the `_selection in ["", "head", "body"]` checks on instantDeath/revive.